redisinsight/ui/src/pages/browser/components/stream-details/messages-view/MessageAckPopover/MessageAckPopover.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { EuiText, EuiPopover, EuiButton } from '@elastic/eui'
+
+import styles from './styles.module.scss'
+
+export interface Props {
+  id: string
+  isOpen: boolean
+  closePopover: () => void
+  showPopover: () => void
+  acknowledge: (entry: string) => void
+}
+
+const AckPopover = ({ id, isOpen, closePopover, showPopover, acknowledge }:Props) => (
```

Comment:
please, destruct inside component
`const { ... } = props`
and add space after `:`